src/ProgressLogging.jl

```diff
@@ -37,7 +192,7 @@ end
 ```
 """
 function progress(f; name = "")
-    _id = gensym()
+    _id = uuid4()
```

Comment:
Earlier we were using `uuid5()` why are we using `uuid4()` here?
Why not `asuuid(name)`?